packages/components/src/Components/TagCount/TagCount.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './tagCount.scss';
+import TagModal from '../TagModal/TagModal'
+
+export default class TagCount extends React.Component
+{
+    constructor(props) {
+        super(props);
+        let count = 0;
+        if (props.systemName === 'paul.localhost.com') {
+            count = 11;
+        } else {
+            count = 2; 
+        }
+        this.state = {
+            modalOpen: false,
+            count: count
+        };
+    }
+
+    toggleTagModal = () => {
+        this.setState(({ modalOpen }) => ({
+            modalOpen: !modalOpen
+          }));
+    }
+
+    render() {
+        return (
+            <div className="ins-c-tagCount" onClick={this.toggleTagModal}>
+                <i className="fas fa-tag"></i>
```

Comment:
you can just have this as a one sided tag

```
 <i className="fas fa-tag"/>
```

Also, patternfly exposes this tag icon as a component. We should probably use that.

Reply:
Yes, definetely use `TagIcon` from `@patternfly/react-cions` instead of `i`.

Reply:
Where do they do that? I don't see it in the icon list in the docs

Reply:
https://patternfly-react.surge.sh/patternfly-4/icons/Icons/

Search for `tag`

Reply:
For some reason, patternfly.org doesn't list all of their icons. Very weird.